src/index.js

```diff
@@ -1,13 +1,10 @@
-var React = require('react');
-var ReactDOM = require('react-dom');
+import App from './components/App';
+import React from 'react';
+import ReactDOM from 'react-dom';
 //var propTypes = require('prop-types');
-require('./index.css');
-var App = require('./components/App');
+import './index.css';
 
-ReactDOM.render(
-	<App />,
-	document.getElementById('app')
-);
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
`root` is what the element name is called in `/public/index.html` (the actual html file that is loaded in the browser). You could change that element id and then change this and it'd work just the same